Ginger/Ginger/ApplicationModelsLib/POMModels/POMEditPage.xaml.cs

```diff
@@ -229,16 +242,50 @@ private void FillTargetAppsComboBox()
             {
                 xTargetApplicationComboBox.ItemsSource = WorkSpace.Instance.Solution.ApplicationPlatforms.Where(x => ApplicationPOMModel.PomSupportedPlatforms.Contains(x.Platform)).ToList();
             }
+        }
 
-            xTargetApplicationComboBox.SelectedValuePath = nameof(ApplicationPlatform.Key);
-            xTargetApplicationComboBox.DisplayMemberPath = nameof(ApplicationPlatform.AppName);
-            CollectionChangedEventManager.AddHandler(source: WorkSpace.Instance.Solution.ApplicationPlatforms, handler: ApplicationPlatforms_CollectionChanged);
+        /// <summary>
+        /// Returns a list of supported agents based on the application platform and agent type.
+        /// </summary>
+        /// <returns>ObservableList of supported agents.</returns>
+        private ObservableList<Agent> SupportedAgents()
+        {
+            ObservableList<Agent> optionalAgentsList;
 
+            var agents = WorkSpace.Instance.SolutionRepository.GetAllRepositoryItems<Agent>();
+
+            if (mAppPlatform == ePlatformType.Web)
+            {
+                List<Agent> list = agents.Where(x => x.Platform == ePlatformType.Web).ToList();
+
+                foreach (var agent in agents)
+                {
+                    if (agent.Platform == ePlatformType.Mobile)
+                    {
+                        var appType = agent.DriverConfiguration.FirstOrDefault(p => string.Equals(p.Parameter, "AppType", StringComparison.OrdinalIgnoreCase))?.Value ?? string.Empty;
+
+                        if (string.Equals(appType, eAppType.Web.ToString(), StringComparison.OrdinalIgnoreCase))
```

Comment:
use nameof(eAppType.Web) instead of eAppType.Web.ToString()

Reply:
done